ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/js/dropdownview.js

```diff
@@ -97,20 +97,25 @@
 
         setModel(model) {
             super.setModel(model);
+            this.#updateModelValue(this.widget);
```

Comment:
shouldn't it be done, if the widget has a value ?

Reply:
I wanted to keep it generic and not introduce any additional if else. in that case, it will set as empty which is corrrect.